snapcraft.yaml

```diff
@@ -37,33 +37,20 @@ apps:
       GSETTINGS_SCHEMA_DIR: $SNAP/share/glib-2.0/schemas
 
 parts:
-# Needed by clutter
-  snapcraft-preload:
-    source: https://github.com/sergiusens/snapcraft-preload.git
-    plugin: cmake
-    build-packages:
-      - gcc-multilib
-      - g++-multilib
-
   lightsoff:
     source: https://gitlab.gnome.org/GNOME/lightsoff.git
     source-type: git
     source-tag: '40.0.1'
-    override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
+    parse-info: [usr/share/metainfo/org.gnome.LightsOff.appdata.xml]
     override-build: |
-      sed -i.bak -e 's|Icon=org.gnome.LightsOff$|Icon=${SNAP}/meta/gui/org.gnome.LightsOff.svg|g' $SNAPCRAFT_PART_SRC/data/org.gnome.LightsOff.desktop.in
-      snapcraftctl build
-      mkdir -p $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp ../src/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $SNAPCRAFT_PART_INSTALL/meta/gui/
-      cp $SNAPCRAFT_PART_BUILD/data/org.gnome.LightsOff.desktop $SNAPCRAFT_PART_INSTALL/meta/gui/
+      sed -i.bak -e 's|Icon=org.gnome.LightsOff$|Icon=${SNAP}/meta/gui/org.gnome.LightsOff.svg|g' $CRAFT_PART_SRC/data/org.gnome.LightsOff.desktop.in
+      craftctl default
+      mkdir -p $CRAFT_PART_INSTALL/meta/gui/
+      cp ../src/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $CRAFT_PART_INSTALL/meta/gui/
```

Comment:
Should we use $CRAFT_PART_SRC here, just to be consistent, like this:
```
cp $CRAFT_PART_SRC/data/icons/hicolor/scalable/apps/org.gnome.LightsOff.svg $CRAFT_PART_INSTALL/meta/gui/
```

Reply:
Done.